podman/domain/images_manager.py

```diff
@@ -333,7 +333,7 @@ def pull(
         return self.resource()
 
     def remove(
-        self, image: str, force: Optional[bool] = None, noprune: bool = False
+        self, image: Union[Image, str], force: Optional[bool] = None, noprune: bool = False
```

Comment:
Maybe this is my ignorance, but shouldn't the "optional" `force` parameter appear last?

Reply:
@cevich Normally I would agree, but the argument order is set for compatibility. While a bit smelly, it is legal because the following arguments have defaults as well.

Reply:
Okay, you certainly understand the compatibility aspects better than I.